### ISSUE TITLE
Use shuffled validation set

### ISSUE THREAD

**Issue by lepisma** [reichlab/neural-stack#11]
Keras' default behavior is to use the _last_ `0.x` fraction for validation. A shuffle of train data as whole would fix this.
